deployment/ccip/sequence/deploy_ccip.go

```diff
@@ -217,90 +215,66 @@ func deployCCIPSequence(b operations.Bundle, deps operation.TonDeps, in DeployCC
-		ExecutorRoleCheckEnabled: true,
-		OpPendingInfo: timelock.OpPendingInfo{
-			ValidAfter:            0,
-			OpFinalizationTimeout: 0,
-			OpPendingID:           big.NewInt(0),
-		},
-		RBAC: rbac.Data{
-			Roles: cell.NewDict(256),
-		},
+	// Invoke deploy Timelock changeset operation
+	deployTimelockInput := mcmsOps.DeployTimelockInput{
+		ContractPath:  tonCompiledContracts[state.Timelock].ContractPath,
+		ID:            in.CCIPConfig.TimelockParams.ID,
+		ChainSelector: in.ChainSelector,
+		Coins:         tonCompiledContracts[state.Timelock].SuggestedTONCoinsForDeployment,
+		MinDelay:      in.CCIPConfig.TimelockParams.MinDelay,
+		Admin:         in.CCIPConfig.TimelockParams.Admin,
+		Proposers:     in.CCIPConfig.TimelockParams.Proposers,
+		Executors:     in.CCIPConfig.TimelockParams.Executors,
+		Cancellers:    in.CCIPConfig.TimelockParams.Cancellers,
+		Bypassers:     in.CCIPConfig.TimelockParams.Bypassers,
 	}
-	timelockMessageBody := timelock.Init{
-		QueryID:                  0,
-		MinDelay:                 in.CCIPConfig.TimelockParams.MinDelay,
-		Admin:                    in.CCIPConfig.TimelockParams.Admin,
-		Proposers:                common.SnakeRef[common.WrappedAddress](common.WrapAddresses(in.CCIPConfig.TimelockParams.Proposers)),
-		Executors:                common.SnakeRef[common.WrappedAddress](common.WrapAddresses(in.CCIPConfig.TimelockParams.Executors)),
-		Cancellers:               common.SnakeRef[common.WrappedAddress](common.WrapAddresses(in.CCIPConfig.TimelockParams.Cancellers)),
-		Bypassers:                common.SnakeRef[common.WrappedAddress](common.WrapAddresses(in.CCIPConfig.TimelockParams.Bypassers)),
-		ExecutorRoleCheckEnabled: true,
-		OpFinalizationTimeout:    0,
+
+	deployTimelockOutput, err := operations.ExecuteOperation(b, mcmsOps.DeployTimelockOp, deps, deployTimelockInput)
+	if err != nil {
+		return output, err
+	} else if !deployTimelockOutput.Output.Address.IsAddrNone() {
+		output.TimelockAddress = newTONContractAddress(deployTimelockOutput.Output.Address, in.ChainSelector, state.Timelock, in.ContractsSemver, in.ContractsVersionSha)
 	}
 
-	err = InvokeDeployContractOperation(b, deps, in.ChainSelector, timelockAddress, tonCompiledContracts[state.Timelock], timelockStorage, timelockMessageBody, func(tonContractAddress *TONContractAddress) {
-		timelockAddress = tonContractAddress.TONAddress
-		output.TimelockAddress = tonContractAddress
-	})
+	// Invoke deploy MCMS changeset operation
+	deployMCMSInput := mcmsOps.DeployMCMSInput{
+		ContractPath:  tonCompiledContracts[state.MCMS].ContractPath,
+		ID:            in.CCIPConfig.MCMSParams.ID,
+		ChainSelector: in.ChainSelector,
+		Coins:         tonCompiledContracts[state.MCMS].SuggestedTONCoinsForDeployment,
+		OpsCount:      in.CCIPConfig.MCMSParams.OpsCount,
+		ChainID:       in.CCIPConfig.MCMSParams.ChainID,
+		PreOpsCount:   in.CCIPConfig.MCMSParams.PreOpsCount,
+		PostOpsCount:  in.CCIPConfig.MCMSParams.PostOpsCount,
```

Comment:
The count values all seem like something that should be initialized to zero? No config